build-scripts/ubuntu-1604/build-3rd-parties.sh

```diff
@@ -83,3 +83,9 @@ build_from_pypi intervaltree 2.1.0
 build_from_pypi jsonpickle 0.9.6
 # TODO: add libsnappy dependency for python-rocksdb package
 build_from_pypi python-rocksdb 0.6.9
+
```

Comment:
Why do you need it here? Even if you need something, there should be a version specified.

Reply:
On a brand new server launched from the plenum docker images.  These dependencies are missing and are required before some of the other dependency packages listed above will install.  These don't have anything to do with the plugin stuff.

Reply:
What kind of Docker images do you use? Is it our officially supported Docker https://github.com/hyperledger/indy-node/tree/master/environment/docker/pool (based on Ubuntu 16)?

Reply:
This is the base Dockerfile that I was testing with
https://github.com/hyperledger/indy-plenum/tree/master/build-scripts/ubuntu-1604/Dockerfile

Reply:
This Docker is just for building deb packages (that is internal CD purposes). 
Please use the link I provided for a correct Docker files to run indy-node in.